File: OCH_Tao/ABC/A/030.js
const input = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(str => parseInt(str, 10));
if (input[1] / input[0] === input[3] / input[2]) {
  console.log("DRAW");
} else if (input[1] / input[0] > input[3] / input[2]) {
  console.log("TAKAHASHI");
} else {
  console.log("AOKI");
}

File: OCH_Tao/ABC/A/030.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  double A, B, C, D;
  cin >> A >> B >> C >> D;
  if (B / A > D / C)
  {
    cout << "TAKAHASHI" << endl;
  }
  else if (B / A < D / C)
  {
    cout << "AOKI" << endl;
  }
  else
  {
    cout << "DRAW" << endl;
  }
}